test/test_migrate.py

```diff
@@ -0,0 +1,60 @@
+"""Test migration pipeline"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+import pandas as pd
+
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+PROJECT_ROOT = TEST_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from src.scripts.migrate import slurp
+
+TEST_INPUT_DIR = TEST_DIR / 'input' / 'test_migrate'
```

Comment:
## Static files selected for testing
Test/Slurp: For `migrate.py` tests, I decided to make special static copies in `test/input/` of the following files:
```py
mondo_mappings_path=str(TEST_INPUT_DIR / 'mondo.sssom.tsv'),
mapping_status_path=str(TEST_INPUT_DIR / 'ordo_mapping_status.tsv'),
```

And not do so for the following files:
```py
ontology_path=str(ONTOLOGY_DIR / 'components' / 'ordo.owl'),
mondo_terms_path=str(ONTOLOGY_DIR / 'tmp' / 'mirror_signature-mondo.tsv'),
onto_config_path=str(ONTOLOGY_DIR / 'metadata' / 'ordo.yml'),
```

My rationale is that the first 2 can be customized so that even when our mapping status with ORDO changes, these static files can be set up to help us test basically every possible edge case we can imagine. 

The other ones haven't been changed because there's not really a need, and if anything goes wrong with those files either, the test should catch that by erroring.

## Possible future changes to static test files
Although, there is some rationale to maybe creating a shortened, static version of `ordo.owl` which will allow the script to run faster.

## Caveats / more possible future changes
I didn't make any dependencies for the `tests` goal, so if some of those files don't exist, it'll error out. Maybe this is a  design flaw. I up the `tests` goal to conveniently automatically find and run all Python unit tests in the project. 

Perhaps a different way would be to split things up, like a `test-slurp` that has goal dependencies, and then a `tests` goal with `test-slurp` as its dependency.